client/src/main/scala/org/apache/celeborn/client/LifecycleManager.scala

```diff
@@ -1583,6 +1590,37 @@ class LifecycleManager(val appUniqueId: String, val conf: CelebornConf) extends
     }
   }
 
+  private def removeBatchExpiredShuffles(): Unit = {
+    val shuffleIds = new util.ArrayList[Integer]()
+    unregisterShuffleTime.keys().asScala.foreach { shuffleId =>
```

Comment:
Thank you for the code review!
I think it would be beneficial to focus solely on the batch processing in this pull request. This way, we can maintain the same time interval and condition as before.
Thanks!